packages/core/src/types.ts

```diff
@@ -2219,3 +2241,55 @@ export type PersistedMachineState<TState extends AnyState> = Pick<
     };
   };
 };
+
+export interface InspectedActorObject {
```

Comment:
it would be nice if all of those inspector-related types would be moved closer to where they are actually used - which currently is the `system.ts`. I find it much easier to navigate the codebase when things stay colocated. Our `./types.ts` here is pretty huge already and I often get lost in it ;p

Reply:
Will move 🚚